docs/sources/configure/options.md

```diff
@@ -1272,6 +1283,18 @@ of Beyla: application-level metrics or network metrics.
   metrics; but only if the Prometheus `port` property is defined. For network-level metrics options visit the
   [network metrics]({{< relref "../network" >}}) configuration documentation.
 
+| YAML                                  | Environment variable                                   | Type     | Default |
+|---------------------------------------|--------------------------------------------------------|----------|---------|
+| `allow_service_graph_self_references` | `BEYLA_PROMETHEUS_ALLOW_SERVICE_GRAPH_SELF_REFERENCES` | boolean  | `false` |
```

Comment:
I'm assuming the YAML name is correct, because it goes on a different scope than the otel one (and thus can have the same name), but just in case I'm noting this here. Feel free to ignore.

---

I'm not sure what's more consistent, but perhaps `BEYLA_PROM_....` might be a tad less verbose and friendlier than having to type `PROMETHEUS`. Not an issue either way, just throwing this out here just in case.

Reply:
We use `PROMETHEUS` for the others, I think we should keep it consistent.